src/styles/components/badges/variables.less

```diff
@@ -6,6 +6,6 @@ badges/variables.less
 --------------------------------------------------------------------------------
 ----------------------------------------------------------------------------- */
 
-
-
 @badges-enabled:                                                                true;
```

Comment:
While we've decided to avoid the comments/spacing in the styles themselves, I'd still love to see it observed in the variables for consistency.  See `menu-tabbed/variables.less` for an example.